src/components/rangeslider/draw.js

```diff
@@ -136,21 +137,31 @@ module.exports = function(gd) {
 
         opts._rl = [range0, range1];
 
+        if(oppAxisRangeOpts.rangemode === 'auto' || oppAxisRangeOpts.rangemode === 'fixed') {
```

Comment:
I'd vote for using `oppAxisRangeOpts.rangemode !== 'match'` instead here and in all the other places below would make thing more readable. Not a big deal though.